crates/bevy_platform_support/src/collections/hash_set.rs

```diff
@@ -0,0 +1,560 @@
+//! Provides [`HashSet`]
```

Comment:
This should call out the origin of this code and why this module exists. Ditto for hash_map.